### PR TITLE
Improve compute nodes database error message

### LINKED ISSUE #11057: Change "Cannot resolve scope id for path" to more clear error
There is an error that often happens when a database can't be resolved by its path because the database was not created due to some problem. When the compute nodes try to start, they get this error and terminate. It is often not obvious to cluster administrators what is going on, because it is hard to find documentation on this error and understand it.

https://github.com/ydb-platform/ydb/blob/7277322ca851564ed92b0ab4aade5fafe2329517/ydb/core/mind/node_broker__register_node.cpp#L63

### PR DESCRIPTION
**Pull request by NeimadTL**
When the compute nodes try to start with a missing database and this error is thrown. However, in order to make it clearer for cluster administrators, the error message has been improved so that they know the reason why the error occurred.

Related issue:  https://github.com/ydb-platform/ydb/issues/11057

### CODE REVIEW

**Comment by NeimadTL** [comment]
Hi @pixcc, would appreciate your review on this, thank you.

**Review by pixcc — CHANGES_REQUESTED** [review]

File: ydb/core/mind/node_broker__register_node.cpp
Comment (NeimadTL) [reply ydb/core/mind/node_broker__register_node.cpp]: Thanks for your feedback :+1:  , just made the change.

**Review by pixcc — APPROVED** [review]

**Comment by pixcc** [comment]
@NeimadTL Thank you for your contribution. Let's wait for the CI checks, after which I'll merge the changes.

**Comment by NeimadTL** [comment]
> @NeimadTL Thank you for your contribution. Let's wait for the CI checks, after which I'll merge the changes.

You're welcome, thanks for your review. Great !

**Pull request merged**
